lua/lvim/core/nvimtree.lua

```diff
@@ -159,7 +159,7 @@ function M.setup()
   end
 
   if lvim.builtin.nvimtree.on_config_done then
-    lvim.builtin.nvimtree.on_config_done(nvim_tree_config)
+    lvim.builtin.nvimtree.on_config_done(nvim_tree)
```

Comment:
can we make sure this happens after line 165? After all, this is on_config done, not on_pre_config done...

Reply:
thanks for pointing that out,fixed it

Reply:
I think it might've had something to do with global variables at the time, since they needed to be called before the setup function itself.